### DISCUSSION TITLE
Русская локализация способностей в AC

### DISCUSSION THREAD

**Discussion by xBarselonx** [trickerer/Trinity-Bots#247] [Q&A]
trickerer, привет!
В AC нет локализованных способностей ботов на русском языке (в TC есть). Хотел бы это сделать. Ранее я уже поднимал этот вопрос https://github.com/trickerer/Trinity-Bots/issues/212#issuecomment-1265148683. 
Где нужно внести изменения, какие таблицы используются? 
Для начала, пока не разберусь, хочу попробовать сделать в своем профиле на копии из твоего репозитория.

**Comment by trickerer** [comment]
https://github.com/trickerer/Trinity-Bots/issues/250
